tests/test.js

```diff
@@ -148,6 +148,16 @@ module.exports.todoMVCTest = function (frameworkName, baseUrl, speedMode, laxMod
 				testOps.assertItemAtIndexIsCompleted(2);
 			});
 
+			test.it('should correctly update the complete all checked state', function () {
+				//manually check all items
```

Comment:
Super nitty: The other comments in the file are introduced with `//` + space. :)